docs/modules/ROOT/pages/release-notes/release-notes-30-0.adoc

```diff
@@ -0,0 +1,34 @@
+= Release Notes - Payara Platform Enterprise 5.30.0
+
+== Supported APIs and Applications
+
+ * Java EE 8 Applications
+ * Jakarta EE 8
+ * MicroProfile 4.1
+
+=== Notes
+==== MicroProfile 4.1
+The release of Payara Enterprise 5.30.0 introduces an update of MicroProfile from 3.3 to 4.1, bringing with it some breaking changes. Details of these changes and how to work around them can be found within the documentation of each spec. 
+
+=== New Features
+* [FISH-1526] - Upgrade Payara Enterprise to MicroProfile 4.1
```

Comment:
Sorry, I wasn't clear - I meant that this should get replaced with the MP 4.1 issue, not just changing the number.
```suggestion
* [FISH-666] - Upgrade Payara Enterprise to MicroProfile 4.1
```